Replace plain text with attributes for content in the platform module (#1119)
ansible/aap-docs: downstream/modules/platform/con-persisting-data-from-auto-runs.adoc

[id="persisting-data-from-auto-runs_{context}"]
= Persisting data from auto runs

> Consider the local {ControllerName} filesystem as counterproductive since that ties the data to that host. If you have a multi-node cluster, then you can contact a different host each time causing issues if you are creating workflows that depend upon each other and created directories. For example, if a directory was only created in one node, while another node runs the playbook, the results would be inconsistent.

Reviewer: ```suggestion
Consider the local {ControllerName} filesystem as counterproductive since that ties the data to that host. If you have a multi-node cluster, then you can contact a different host each time, causing issues if you are creating workflows that depend on each other and created directories. For example, if a directory was only created by one node while another node runs the playbook, the results would be inconsistent.
```

The solution is to use some form of shared storage solution, such as Amazon S3, Gist, or a role to rsync data to your data endpoint. 

The option exists of injecting data or a configuration into a container at runtime. This can be achieved by using the {ControllerName}'s isolated jobs path option.

> This provides a way to mount directories and files into an {ExecEnvShort} at runtime. This is achieved through the {AutomationMesh}, utilizing ansible-runner to inject them into a Podman container to start the automation. What follows are some of the use cases for using isolated job paths:

Reviewer: Replace "utilizing" with "using" as it is a word to avoid in SG.

Author: Thanks for flagging this. 
It doesn't seem to be catered for in the Vale rules so I'll take a look at that 👍
